nuxeo-ai-core/src/test/java/org/nuxeo/ai/rest/TestRestEnrichment.java

```diff
@@ -45,13 +48,19 @@
 import org.nuxeo.runtime.test.runner.Deploy;
 import org.nuxeo.runtime.test.runner.Features;
 import org.nuxeo.runtime.test.runner.FeaturesRunner;
+
 import com.fasterxml.jackson.databind.JsonNode;
+import com.github.tomakehurst.wiremock.extension.responsetemplating.ResponseTemplateTransformer;
+import com.github.tomakehurst.wiremock.junit.WireMockRule;
 
 @RunWith(FeaturesRunner.class)
-@Features({EnrichmentTestFeature.class, PlatformFeature.class})
-@Deploy({"org.nuxeo.ai.ai-core:OSGI-INF/enrichment-rest-test.xml"})
+@Features({ EnrichmentTestFeature.class, PlatformFeature.class })
+@Deploy({ "org.nuxeo.ai.ai-core:OSGI-INF/enrichment-rest-test.xml" })
 public class TestRestEnrichment {
 
+    @Rule
+    public WireMockRule wmr = new WireMockRule(options().extensions(new ResponseTemplateTransformer(true)).port(5089));
```

Comment:
`explorer.nuxeo.com` started to return `500`, therefore breaking tests. It shouldn't be used in first place but history remains history